src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax;
+
+import core.basesyntax.models.*;
+
+public class Main {
+    public static void main(String[] agrs) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < 3; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (int i = 3; i < 6; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
+        }
+
+        for (Figure figure : figures) {
+            System.out.println("Figure: " + getFigureName(figure)
+                    + ", area: " + figure.getArea()
+                    + " sq. units"
+                    + getFigureDescription(figure)
+                    + ", color: " + getColor(figure));
+        }
+    }
+
+    private static String getFigureName(Figure figure) {

```

Comment:
Avoid using static methods in your solution. Here, `getFigureName`, `getFigureDescription`, and `getColor` are static utility methods which could be instance methods in their respective classes. Consider refactoring these into the appropriate classes as instance methods, which will be more object-oriented.